### ISSUE TITLE
Error installing SwiftyDropbox

### ISSUE THREAD

**Issue by vikasbnsl** [dropbox/SwiftyDropbox#32]
Getting following error while installing SwiftyDropbox from cocoa pods

[!] Error installing SwiftyDropbox
[!] /usr/bin/git clone git@github.com:dropbox/SwiftyDropbox.git /var/folders/ts/hz4_l6ks6tl8tcgjsvmb2tcc0000gn/T/d20151118-1772-sdgeje --single-branch --depth 1 --branch 0.5

Cloning into '/var/folders/ts/hz4_l6ks6tl8tcgjsvmb2tcc0000gn/T/d20151118-1772-sdgeje'...
Warning: Permanently added the RSA host key for IP address '192.30.252.128' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by smarx** [comment]
Take a look at https://help.github.com/articles/error-permission-denied-publickey/ to get a public key set up with your GitHub account.

But I think this can also be fixed by using an HTTPS URL in the Podfile. Would you mind giving this a try?

```
platform :ios, '8.0'
use_frameworks!

pod 'SwiftyDropbox', :git => 'https://github.com/dropbox/SwiftyDropbox.git', :tag => '0.8'
```

I believe this will fetch the repository unauthenticated over HTTPS instead of trying to use your GitHub account (which you may not have set up).

**Comment by vikasbnsl** [comment]
great!! changing URL worked like a charm "
https://github.com/dropbox/SwiftyDropbox.git"

thanks a ton.

On 19 November 2015 at 06:56, Ryan Pearl <notifications@github.com> wrote:

> Closed #32 <https://github.com/dropbox/SwiftyDropbox/issues/32>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dropbox/SwiftyDropbox/issues/32#event-468672391>.
>



-- 

Regards,

*Vikas Bansal*

Mobile: +91 8802339189
Email:   *bansal.vks@gmail.com* <bansal.vks@gmail.com>
Social:  [image: Facebook] <http://www.facebook.com/100001657761600>  [image:
Google Plus] <https://plus.google.com/+VIKASBANSALji>
